apps/web-giddh/src/app/contact/contact.component.scss

```diff
@@ -341,14 +326,14 @@ span.inputAddon {
     .onDesktopView.btn-group {
         display: none;
     }
-    .custumerRightWrap {
+    .customer-right-wrap {
         padding-right: 0;
         position: fixed;
         bottom: 0;
         width: 100%;
         background: #fff;
```

Comment:
use variable for         background: #fff;

Reply:
not worked on particular line, please focus on red and green color line